recipe/meta.yaml

```
@@ -1,60 +1,57 @@
 {% set name = "pydantic" %}
-{% set version = "2.5.3" %}
-{% set repo_url = "https://github.com/pydantic/pydantic" %}
-{% set docs_url = "https://docs.pydantic.dev/latest/" %}
+{% set version = "2.8.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b3ef57c62535b0941697cce638c08900d87fcb67e29cfa99e8a68f747f393f7a
-  patches:
-    - 0001-do-not-depend-on-hatch-fancy-pypi-readme.patch
+  sha256: 6f62c13d067b0755ad1c21a34bdd06c0c12625a22b0fc09c6b149816604f7c2a
+
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-  skip: true  # [py<37]
+  skip: true  # [py<38]
 
 requirements:
-  build:
-    - patch     # [not win]
-    - m2-patch  # [win]
   host:
     - python
     - pip
     - hatchling
+    - hatch-fancy-pypi-readme >=22.5.0
   run:
     - python
     - typing-extensions >=4.6.1
     - annotated-types >=0.4.0
-    - pydantic-core ==2.14.6
+    - pydantic-core ==2.20.1
+    - tzdata  # [py>=39]
+
   run_constrained:
     - email-validator >=2.0.0
 
 test:
   imports:
-    - {{ name }}
+    - pydantic
   requires:
     - pip
   commands:
     - pip check
 
 about:
-  home: {{ repo_url }}
+  home: https://github.com/pydantic/pydantic
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: Data validation and settings management using python type hinting
   description: |
     Data validation and settings management using python type hinting.
-    See documentation <{{ docs_url }}> for more details.
-  doc_url: {{ docs_url }}
-  dev_url: {{ repo_url }}
+    See documentation doc_url for more details.
+  doc_url: https://docs.pydantic.dev/latest/
+  dev_url: https://github.com/pydantic/pydantic
 
 extra:
   recipe-maintainers:
     - samuelcolvin
-    - dgasmith
+    - dgasmith
```

```suggestion
    - dgasmith

```
Missing an empty ending line